docs/guides/code-server.md

```diff
@@ -107,21 +109,26 @@ Finally, now that we've got Go all we need to go is grab the latest source and b
 
 ```bash
 # get the latest source
-$ git clone https://github.com/pomerium/pomerium.git
-# grab make
-$ sudo apt-get install make
+git clone https://github.com/pomerium/pomerium.git
+
 # build pomerium
-$ make build
+cd pomerium && make build
 # run pomerium!
-$ ./bin/pomerium --version
-v0.2.0+e1c00b1
+./bin/pomerium --version
+# v0.2.0+e1c00b1
```

Comment:
FYI this failed for me inside code-server. I was planning on updating the version number

```sh
coder@929f14a68fd0:~/project/pomerium$ make build
==> build
./scripts/get-envoy.bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   625  100   625    0     0   3033      0 --:--:-- --:--:-- --:--:--  3033
100 58.9M  100 58.9M    0     0  7347k      0  0:00:08  0:00:08 --:--:-- 8291k
build github.com/pomerium/pomerium/cmd/pomerium: cannot load crypto/ed25519: cannot find module providing package crypto/ed25519
make: *** [Makefile:90: build] Error 1
```

Reply:
(the other packages were installed, this was the second time running to get the error again)

Reply:
I added some suggestions. I think (hope?) that the updated version of Go should solve that issue.